tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/notificator/DiscordNotificator.java

```diff
@@ -319,8 +334,8 @@ public void notifyBuildFailing(@NotNull SRunningBuild sRunningBuild, @NotNull Se
     @Override
     public void notifyBuildProbablyHanging(@NotNull SRunningBuild sRunningBuild, @NotNull Set<SUser> users) {
         String title = "Build is probably hanging";
-        String description = "The build with the ID " + sRunningBuild.getBuildId() + " is probably hanging!";
-        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + sRunningBuild.getBuildId();
+        String description = "The build with the ID " + sRunningBuild.getBuildNumber() + " is probably hanging!";
+        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + sRunningBuild.getBuildNumber();
```

Comment:
Please continue to use `sRunningBuild.getBuildId();` here.
The URL expects the build id. If the build number is customized in the settings, the URL won't work anymore.